src/app/pages/explore-container/explore-container.component.html

```diff
@@ -1,5 +1,8 @@
 <ion-header class="ion-no-border">
     <ion-toolbar mode="md" #toolbar style="--background: transparent;">
+        <ion-buttons slot="start">
+            <ion-img src="/assets/imgs/icon.png" style="width: 25px; padding-left: 7px;"></ion-img>
```

Comment:
I would add a white background to better match the logo of the app. This would also to add a bit of contrast with all those shades of blue. 

Tried this:

```css
.beping-icon {
  width: 2rem;
  height: auto;
  background-color: #ffffff;
  border-radius: 50%;
  padding: 0.4rem;
  margin-left: 10px;
}
ion-header > ion-toolbar > ion-title {
  padding-inline: 10px;
}
```

light theme:
![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/13313104/163700586-9fffff81-ec2f-45ea-bcb9-c9ea5ddd3b87.gif)

Dark theme:
![ezgif com-gif-maker](https://user-images.githubusercontent.com/13313104/163700584-8bd278f7-ca06-491d-8386-8eb93892f136.gif)

WDYT?

Reply:
Sorry I get to this earlier. That's a great addition!